src/components/map/layers/RoutePathLinkLayer.tsx

```diff
@@ -2,11 +2,11 @@ import React, { Component } from 'react';
 import * as L from 'leaflet';
 import { Polyline, FeatureGroup } from 'react-leaflet';
 import { observer, inject } from 'mobx-react';
-import { IRoutePathLink } from '~/models';
+import { INode, IRoutePathLink } from '~/models';
 import NodeType from '~/enums/nodeType';
 import { PopupStore } from '~/stores/popupStore';
-import NodeLayer from './NodeLayer';
-import NodeMarker from './NodeMarker';
+import NodeMarker from '~/components/map/layers/objects/NodeMarker';
```

Comment:
These two components are from the same folder, but imported in different ways. I don't want to set rules on which way to import, but I think in these cases the fact that they are located in the same folder, could be clearer, if they were imported in the same way.